lib/dnsimple/struct/certificate.rb

```diff
@@ -8,9 +8,15 @@ class Certificate < Base
       # @return [Integer] The associated domain ID.
       attr_accessor :domain_id
 
+      # @return [String] The certificate name.
```

Comment:
I left the name out on purpose, as it's an internal representation. It's also misleading (as it depends on the domain name).

As you have the alternate names, and the common name, I would not expose the name.